src/initializers/joi.ts

```diff
@@ -171,14 +171,6 @@ const Joi: JoiWithExtensions = _Joi.extend(
   joi => ({
     type: 'safeHtml',
     base: joi.string(),
-    rules: {
-      allowEmpty: {
-        method() {
-          // @ts-ignore
-          return this.$_root.allow('').allow(null);
```

Comment:
❓ Why was this buggy?

Reply:
This overrides the whole scema, an the `prepare` method is not called.

In addition, `allowEmpty` can be achieved with `.allow('', null)`

Reply:
Thanks for the explanation